README.rst

```diff
@@ -361,6 +362,42 @@ Note that ``THREADS`` backend is not recommended since `Kokkos does not guarante
 
 .. installation_LKokkos-end-inclusion-marker-do-not-remove
 
+.. installation_LTensor-start-inclusion-marker-do-not-remove
+
+Lightning-Tensor installation
+*****************************
+Lightning-Tensor requires CUDA 12 and the `cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`_ (only the `cutensornet <https://docs.nvidia.com/cuda/cuquantum/latest/cutensornet/index.html>`_ library is required).
+The SDK may be installed within the Python environment ``site-packages`` directory using ``pip`` or ``conda`` or the SDK library path appended to the ``LD_LIBRARY_PATH`` environment variable.
+Please see the `cuQuantum SDK <https://developer.nvidia.com/cuquantum-sdk>`_ install guide for more information.
+
+Install Lightning-Tensor from source
+====================================
+
+To install Lightning-Tensor from the package sources using the direct SDK path, Lightning-Qubit should be installed before Lightning-Tensor:
+
+.. code-block:: console
+
+    git clone https://github.com/PennyLaneAI/pennylane-lightning.git
+    cd pennylane-lightning
+    pip install -r requirements.txt
+    PL_BACKEND="lightning_qubit" pip install -e .
```

Comment:
I wonder whether we should remove `-e` flags throughout since this is instructing users as well as developers.  
```suggestion
    PL_BACKEND="lightning_qubit" pip install .
```